flashinfer/aot.py

```diff
@@ -0,0 +1,504 @@
+import argparse
+import os
+import shutil
+from itertools import product
+from pathlib import Path
+from typing import List, Tuple
+
+import torch
+from torch.utils.cpp_extension import _get_cuda_arch_flags
+
+from .activation import act_func_def_str, gen_act_and_mul_module
+from .cascade import gen_cascade_module
+from .custom_all_reduce import gen_comm_module
+from .gemm import gen_gemm_module, gen_gemm_sm90_module, gen_gemm_sm100_module
+from .jit import JitSpec, build_jit_specs
+from .jit import env as jit_env
+from .jit import (
+    gen_batch_decode_module,
+    gen_batch_mla_module,
+    gen_batch_prefill_module,
+    gen_fmha_cutlass_sm100a_module,
+    gen_single_decode_module,
+    gen_single_prefill_module,
+)
+from .norm import gen_norm_module
+from .page import gen_page_module
+from .quantization import gen_quantization_module
+from .rope import gen_rope_module
+from .sampling import gen_sampling_module
+
+
+def gen_gqa_fa2(
```

Comment:
why calling it "gqa"?

Reply:
Oh, I thought because it can handle MHA, MQA, GQA. Would you mind proposing another name?